src/engine/client/client.h

```diff
@@ -544,15 +544,15 @@ void CL_Record(std::string demo_name);
 //
 // cl_keys (for input usage)
```

Comment:
Let's remove the Key_* prefix and put all new keyboard code inside the Keyboard namespace.